planetscale/databases.go

```diff
@@ -14,6 +14,7 @@ type CreateDatabaseRequest struct {
 	Organization string
 	Name         string `json:"name"`
 	Notes        string `json:"notes"`
+	Region       string `json:"region,omitempty"`
```

Comment:
We `omitempty` here because by default, we will create a database in `us-east` if this is unset. If we want to be more explicit with the contract and ensure a valid region is always set, we should probably tweak this.